docs/content/development/rest/_index.md

```diff
@@ -25,7 +25,7 @@ The CDS UI uses a non-persistent token. If you want to play with CDS API, you pr
 You can generate it with:
 
 - [cdsctl login]({{< relref "/docs/components/cdsctl/login.md" >}})
-- Code it with the [Go SDK]({{< relref "/development/sdk/golang/_index.md" >}})
+- Code it with the [Go SDK]({{< relref "/development/sdk/golang.md" >}})
```

Comment:
all of this is wrong, there is a pending task about authentication in the PR #4556